pages/patient-information/index.tsx

```diff
@@ -29,17 +28,15 @@ const PatientInformationPage = () => {
     resolver: zodResolver(schema),
   });
 
-  const [values, setValues] = useState<FormSchemaType | null>(null);
+  const router = useRouter();
 
-  useQuery(["complete-profile", values], () => completeProfile(values), {
-    enabled: !!values,
+  const mutateComplete = useMutation({
```

Comment:
mutation, not mutate. This is an object therefore it should have a noun and not a verb